config/environments/test.rb

```diff
@@ -69,7 +69,7 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
 
-  config.cache_store = :file_store, Rails.root.join("tmp", "cache", "paralleltests#{ENV['TEST_ENV_NUMBER']}")
+  config.cache_store = :null_store
```

Comment:
This was introduced as part of the parallel testing feature. We never used *actual* caching in tests before, and if we did *test* caches, we used mocked as far as I can tell so we should rely on the null store instead.